std_msgs/Header header
int16 left_x
int16 left_y
int16 right_x
int16 right_y
bool button_a
bool button_b
bool button_x
bool button_y
bool button_lb
bool button_rb
bool button_back
bool button_start
bool button_home
bool dpad_up
bool dpad_down
bool dpad_left
bool dpad_right
